btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Dropdown.vue

```diff
@@ -48,66 +49,64 @@
         <UIcon class="float-right text-2xl " name="i-mdi-chevron-down" />
       </ComboboxButton>
       <div v-show="open" class="absolute z-10 w-full">
-        <ul>
```

Comment:
What accessibility error does it fix ?

Reply:
The error message: "Fix all of the following:\n  List element has direct children that are not allowed: [role=combobox], [role=listbox]".   I think it is not necessary to use <ul> and <li> here.